### ISSUE TITLE
Related to computing test accuracy

### ISSUE THREAD

**Issue by PrateekMunjal** [sinhasam/vaal#2]
Congrats for nice paper, it was a very well read! I have doubt in function test of solver class i.e while computing the test accuracy I do not observe the task_model in eval mode. Now as the task_model is vgg16_bn which has batchnorm and dropout layers, so it is must to make the task_model in eval mode before testing. Kindly let me know, if I am wrong somewhere in understanding your code.

**Comment by sinhasam** [comment]
Yes, you are completely correct about the issue! vgg has both bn and dropout, so doing a `task_model.eval()` is correct. I have fixed the issue by adding the line of code. thank you for pointing it out!
